libs/application/templates/mortgage-certificate/src/fields/PendingRejectedTryAgain/index.tsx

```diff
@@ -100,33 +108,23 @@ export const PendingRejectedTryAgain: FC<FieldBaseProps> = ({
       >
         <Text fontWeight="semiBold">Valin fasteign</Text>
         <Text>
-          {selectedProperty?.propertyNumber}{' '}
-          {selectedProperty?.defaultAddress?.display}
+          {propertyDetails?.propertyNumber}{' '}
+          {propertyDetails?.defaultAddress?.display}
         </Text>
       </Box>
-      <Box paddingBottom={3} hidden={loading || !showErrorMsg}>
-        <AlertMessage
-          type="error"
-          title="Ekki gekk að sækja vottorð fyrir þessa eign"
-          message="Vinsamlega hafðu samband við sýslumann, það er búið að senda inn beiðni um leiðréttingu"
-        />
-      </Box>
-
-      <Divider />
-      <Box
-        paddingTop={5}
-        paddingBottom={5}
-        justifyContent="flexEnd"
-        display="flex"
-      >
-        <Button
-          onClick={() => handleClickValidateMortgageCertificate()}
-          disabled={loading}
-          icon="arrowForward"
-        >
-          {formatMessage(m.continue)}
-        </Button>
-      </Box>
+      {!loading && showErrorMsg ? (
+        <Box paddingBottom={3}>
+          <AlertMessage
+            type="error"
+            title="Ekki gekk að sækja vottorð fyrir þessa eign"
+            message="Vinsamlega hafðu samband við sýslumann, það er búið að senda inn beiðni um leiðréttingu"
```

Comment:
Messages.ts?

Reply:
@johannaagma Shouldn't this message be stored in the messages file for easier translation down-the-line?